src/views/name.njk

```diff
@@ -1,6 +1,6 @@
 {% extends "layout.njk" %}
 
-{% block content %}
+{% set partnershipType = props.parameters['which-type'] %}
```

Comment:
this one has been removed
must be deleted

Reply:
Deleted.